en/django_forms/README.md

```diff
@@ -196,7 +196,7 @@ It's time to fill in the dots `[...]`. If `method` is `POST` then we want to con
 form = PostForm(request.POST)
 ```
 
-The next thing is to check if the form is correct (all required fields are set and no incorrect values have been submitted). We do that with `form.is_valid()`.
+The next thing is to check if the form is correct. (All required fields are set and no incorrect values have been submitted.) We do that with `form.is_valid()`.
```

Comment:
IMHO the previous version was clearer. It's an explanation of correct and this decouples it.

Reply:
I agree with ekohl here.